### PR TITLE
Jenkinsfile: temporarily pin windows image to 10.0.17763.973

### PR DESCRIPTION
**Pull request by thaJeztah**
The latest `ltsc2019` image (`10.0.17763.1039`) appear to be broken,
and even a `RUN Write-Host hello` hangs.

Temporarily switching back to an older version so that CI doesn't fail.

### CODE REVIEW

**Comment by thaJeztah** [comment]
Windows stage looks to be green with this change @StefanScherer 

(linux failure is a flaky test)

**Review by StefanScherer — APPROVED** [review]
LGTM

**Comment by StefanScherer** [comment]
Let's go with the previous tag, CI to rebuild the VM images is in progress due to some GitHub org refactorings.

**Comment by thaJeztah** [comment]
moved it out of draft, and kicked CI again to see if the Linux stage goes green (but any changes in Linux should be unrelated)

**Review by AkihiroSuda — APPROVED** [review]

**Comment by cpuguy83** [comment]
> Let's go with the previous tag, CI to rebuild the VM images is in progress due to some GitHub org refactorings.

@StefanScherer Do you mean we shouldn't merge this?

**Comment by thaJeztah** [comment]
@cpuguy83 no (discussed on slack); @StefanScherer was working on updating the workers and see if that would resolve the issue, but this is taking longer, so in the meantime downgrading the image ("previous tag") works.


After this is merged, I'll prepare a draft "revert" PR so that we can test once the upgraded workers are available

**Review by cpuguy83 — APPROVED** [review]
Oh sure, I believe you 😄 

LGTM

**Comment by thaJeztah** [comment]
> Oh sure, I believe you 😄

passed-the-test

**Review by tiborvass — APPROVED** [review]

**Comment by StefanScherer** [comment]
ping @thaJeztah we can revert this temporary fix, the `windows-2019` Jenkins agents are updated to `10.0.17763.1039`, and running containers now works with the latest servercore image which I have tested in our [canaries](https://ci-next.docker.com/teams-canaries/job/test-2019-problem/6/console) environment.

**Comment by thaJeztah** [comment]
Perfect, opened https://github.com/moby/moby/pull/40512

**Comment by thaJeztah** [comment]
This one is related to https://github.com/moby/moby/issues/40525#issuecomment-587993478

> Please see:  [https://support.microsoft.com/en-us/help/4542617/you-may-encounter-issues-when-using-windows-server-docker-container-im](https://support.microsoft.com/en-us/help/4542617/you-may-encounter-issues-when-using-windows-server-docker-container-im) for the latest status.

**Pull request merged**
